PoshBot/Plugins/Builtin/Public/Slap.ps1

```diff
@@ -0,0 +1,23 @@
+function Slap {
+    <#
+    .SYNOPSIS
+        Slap a user with a large trout
+    .PARAMETER User
+        The user that will be slapped
+    .EXAMPLE
+        !slap --user jaap
+    #>
+
+
+    [PoshBot.BotCommand(Permissions = 'view')]

```

Comment:
Do we need a permission on this command? I'd imagine anyone should be able to invoke this in which case `[PoshBot.BotCommand(Permissions = 'view')]` isn't needed.